### PR TITLE
relation input titles

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#17789` add titles and en translations`](http://community.openproject.org/work_packages/17789)
- [x] Someone needs to set the German translation and check the English translations.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-feature-17789-relation-input-titles.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-feature-17789-relation-input-titles.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/public/templates/work_packages/tabs/_add_work_package_relation.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     id="relation_to_id-{{ handler.relationsId }}"
     name="relation[to_id][{{ handler.relationsId }}]"
     ng-model="$parent.relationToAddId"
+    title="{{ I18n.t('js.relation_button_inputs.' + handler.relationsId) }}"
```
Comment (myabc) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @0xF013 can we not reuse `js.relation_labels` instead?

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @myabc if the German translation is suitable and if we don't need some more eloquent text like "start typing for {{name}} task autocomplete"

**Comment by 0xF013** [comment]
@myabc should I switch to `js.relation_labels` ?

File: frontend/public/templates/work_packages/tabs/_add_work_package_relation.html (hunk 1 above)
Comment (myabc) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @0xF013 This is pretty generic functionality. I'd suggest we add one key for field prompt (is this the right terminology?) and interpolate the name of the relation.

``` js
// en.js.field_value_enter_prompt: "Enter a value for '%s'"
// de.js.field_value_enter_prompt: "Einen Wert für '%s' eingeben"
{{ I18n.t('field_value_enter_prompt' , { field: I18n.t('js.relation_labels.' + handler.relationsId) }) }}
```

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @myabc  relation_labels does not contain the same values as relationsId

Comment (myabc) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @0xF013 Can we not just make rename the translation keys to conform to relationsId? (I think it's only one key that's actually different)

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/_add_work_package_relation.html]: @myabc pushed

**Comment by myabc** [comment]
@0xF013 :+1:

**Pull request merged**